ui/types/vault/models/pki/certificate/generate.d.ts

```diff
@@ -10,4 +10,5 @@ export default class PkiCertificateGenerateModel extends Model {
   formFieldsGroup: {
     [k: string]: FormField[];
   }[];
+  validate(): ModelValidations;
```

Comment:
you're a typescript natural!

Reply:
haha i was stuck on this for a little bit! and realized there was a type file 😅